Sources/FluentKit/Concurrency/Database+Concurrency.swift

```diff
@@ -0,0 +1,15 @@
+#if compiler(>=5.5) && $AsyncAwait
+import _NIOConcurrency
+
+@available(macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, *)
+public extension Database {
+    func transaction<T>(_ closure: @escaping (Database) -> EventLoopFuture<T>) async throws -> T {
```

Comment:
Shouldn't the closure be `_ closure: @escaping (Database) async throws -> T` as well?

Reply:
Oops! Yep it should be. See #439